plugins/jobs/src/js/data/JobModel.ts

```diff
@@ -141,7 +156,31 @@ export const resolvers = ({
       args: GeneralArgs,
       _context = {}
     ): Observable<JobLink> {
-      return runJob(args.id).map(({ jobId }) => ({ jobId }));
+      if (args.id) {
+        return runJob(args.id).map(({ jobId }) => ({ jobId }));
+      }
+
+      return Observable.throw({
```

Comment:
I think we need to talk about what we want to throw as a team. I in the other resolvers we just throw strings and I would like to see it concise, but I think having an object with additional info is also nice 🤔

Reply:
I agree, and I would love to discuss that here. But changing it I believe would be out of the scope since I am following the API all the other resolvers use, and I think we should fix them all together. 

I think it is maybe a topic for the patterns we wanna have with the data layer.

Reply:
I think we missed that already and we are unaligned already: https://github.com/dcos/dcos-ui/blob/8429c574a2a99c7704127ee4029a90f69302df55/plugins/jobs/src/js/data/JobModel.ts#L99

Reply:
I have no real preference between the `String` and the `response.mesage.String`. if you have a preference I would use it.